web/packages/core/src/shadow-template.ts

```diff
@@ -77,6 +77,7 @@ export function applyStaticStyles(styleElement: HTMLStyleElement) {
 
         `#container {
             overflow: hidden;
+            outline: none;
```

Comment:
This change now needs to be made here:
https://github.com/ruffle-rs/ruffle/blob/e25d97195eb6a2890d4015df9916ef16f0e1ed37/web/packages/core/src/internal/ui/static-styles.tsx#L50

Reply:
Thanks!

Reply:
Now it's in the .css file
Sorry :D

Reply:
<image src="https://github.com/user-attachments/assets/eeac9487-cb89-4ff2-815a-fc10523c6094" width="300"/>